File: packages/expo-crypto/android/src/main/java/expo/modules/crypto/CryptoPackage.java

package expo.modules.crypto;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import expo.modules.core.BasePackage;
import expo.modules.core.ExportedModule;
import expo.modules.core.ViewManager;

public class CryptoPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new CryptoModule(context));
  }
}

File: packages/expo-crypto/android/src/main/java/expo/modules/crypto/CryptoPackage.kt
package expo.modules.crypto

import android.content.Context
import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class CryptoPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> = listOf(CryptoModule(context))
}
